Formula/ocp.rb

```diff
@@ -31,19 +31,20 @@ class Ocp < Formula
   depends_on "libvorbis"
   depends_on "mad"
 
-  if MacOS.version < :catalina
-    depends_on "sdl"
-  else
-    depends_on "sdl2"
-  end
-
   uses_from_macos "ncurses"
   uses_from_macos "zlib"
 
   on_linux do
     depends_on "util-linux" => :build # for `hexdump`
   end
 
+  on_catalina :or_newer do
+    depends_on "sdl2"
+  end
+  on_mojave :or_older do
+    depends_on "sdl"
+  end
```

Comment:
I'm guessing current Linux bottle was also built with `sdl` due to original conditional.